services/provider/proto/provider.proto

```diff
@@ -161,4 +161,7 @@ message ReportStatusRequest{
     string clientOperatorVersion = 3;
 }
 
-message ReportStatusResponse{}
+message ReportStatusResponse{
+    // Desired version which provider operator wants client operator to match
+    string providerOperatorVersion = 1;
```

Comment:
Can we be more specific here? 
ODFVersion? OCSVersion?

Reply:
Actually, I am not sure why the client needs to know anything about provider versions. This should be framed as a desired version not a provider component version.

So maybe `clientDesiredVersion` would work best

Reply:
done.